.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.155.1/containers/docker-existing-dockerfile
+{
+	"name": "Vue front dev",
+	"dockerComposeFile": [
+		"../docker-compose.yml"
+	],
+	"service": "app",
+	"workspaceFolder": "/app",
+	// Set *default* container specific settings.json values on container create.
+	"settings": { 
+		"terminal.integrated.shell.linux": null,
+		"launch": {
+			"version": "0.2.0",
+			"configurations": [
+				{
+					"jinja": true
```

Comment:
wyczytałem w prezentacji z wykładu, że to w sumie jest typowo pythonowe, wiec w vuejs jest chyba niepotrzebne